File: BECore/app/model/ProjectProgressReport.js
/// <summary>
/// Model EXTJS ProjectProgressReport.
/// </summary>
Ext.define('QLDT.model.ProjectProgressReport', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectProgressReportID',
    idType: 'Guid',
    fields: [
		{ name: "ProjectProgressReportID", type: "string" },
		{ name: "ProjectID", type: "string" },
		{ name: "TermID", type: "int" },
		{ name: "TermName", type: "string" },
		{ name: "DateReport", type: "date" },
		{ name: "DateCheck", type: "date" },
		{ name: "Result", type: "int" },
        { name: "Result_sTen", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: 'IdFiles', type: 'string' }
    ]
});

File: MT.Model/Business/ProjectProgressReport.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectProgressReport:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectProgressReport:EntityBase.
    /// </summary>
    public class ProjectProgressReport : EntityBase
    {
        [Key]
        public Guid ProjectProgressReportID { get; set; }
        public Guid ProjectID { get; set; }
        public int TermID { get; set; }
        public string TermName { get; set; }
        public DateTime DateReport { get; set; }
        public DateTime DateCheck { get; set; }
        public int Result { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
        /// <summary>
        /// Danh sách file đính kèm
        /// </summary>
        /// Create by: truongnm:25.02.2018
        public List<ProjectProgressReport_AttachDetail> Details { get; set; }

        public string Result_sTen { get; set; }

        public string IdFiles { get; set; }
    }
    #endregion
}

